### ISSUE TITLE
part 8 - test for twitterfeed

### ISSUE THREAD

**Issue by Lechus** [daylerees/dependency-injection-example#6]
Hi,
I'm newbie to TDD so my appologize for that question, but shouldn't we write test for Twitter like for Facebook?
`public function testSocialFeedCanBeInstantiated()
    {
        $t = M::mock('Example\FeedReaders\FacebookFeedReader');
        $s = new SocialFeed($t);
    }`

`public function testSocialFeedCanBeInstantiated2()
    {
        $t = M::mock('Example\FeedReaders\TwitterFeedReader');
        $s = new SocialFeed($t);
    }`

**Comment by youhackme** [comment]
No, we should not. Why? Because they are third-party libraries with their own test suites.

**Comment by youhackme** [comment]
whoops just realised this is a really old issue!
